model/date/ui/src/lib/DateSystemConfigurationComponent.svelte

```diff
@@ -5,7 +5,37 @@
 
     export let systemConfiguration: Writable<SystemConfiguration>
     console.log({systemConfiguration})
+
+    let dateFormatObj = [
+        { key: "ddd MMM DD HH:mm:ss Z YYYY", value:"ddd MMM DD HH:mm:ss Z YYYY (Sun Jan 22 23:04:05 -0000 2006)"},
+        { key: "MMMM DD YYYY HH:mm:ss", value: "MMMM DD YYYY HH:mm:ss (January 22 2006 23:04:05)" },
+        { key: "MMMM DD YYYY", value:"MMMM DD YYYY (January 22 2006)" },
+        { key: "YYYY-MM-DDTHH:mm:ssZ", value: "YYYY-MM-DDTHH:mm:ssZ (2006-01-22T23:04:05-0000)" },
+        { key: "YYYY-MM-DD HH:mm:ss Z", value: "YYYY-MM-DD HH:mm:ss Z (2006-01-22 23:04:05 -0000)" },
+        { key: "YYYY-MM-DD", value: "YYYY-MM-DD (2006-01-22)" },
+        { key: "MM/DD/YYYY", value: "MM/DD/YYYY (01/22/2006)" },
+        { key: "MM/DD/YY", value: "MM/DD/YY (01/22/06)" },
+        { key: "DD-MM-YYYY", value: "DD-MM-YYYY (22-01-2006)" },
+        { key: "DD/MM/YYYY", value: "DD/MM/YYYY (22/01/2006)" },
+        { key: "DD/MM/YY", value: "DD/MM/YY (22/01/06)" },
+        { key: "X", value: "X (1137971045)" }
+    ]
+  
```

Comment:
Great that we are making a drop down.  But is this too complicated?  Check what Airtable does .  5 or 6 simple options, with a live example so you can see what it looks like.  

Also, the current date in cozemble does not support time.  My thinking so far was to model time as its own type....but not sure if that is a good idea.

Reply:
Let's discuss this, I'm willing to make adjustments.